lib/runner/VirtualDeviceInvoker.js

```diff
@@ -65,14 +65,18 @@ module.exports = class VirtualDeviceInvoker extends Invoker {
 
             messages.push(message);
         }
+        messages.push({
```

Comment:
Please call https://github.com/bespoken/virtual-device-sdk/blob/master/src/VirtualDevice.ts#L145 instead - this is our standard way to wait for the session to conclude.